### PR TITLE
transactionsRoles: from/to matching to be conjunctive (AND) rather than disjunctive (OR)

### LINKED ISSUE #80: transactionsRoles with to and from fields not returning correct transactions
transactionsRoles with the 'to' and 'from' fields specified is returning transactions that satisfy either the 'to' address or the 'from' address or satisfy both. It should return just transactions that satisfy both the 'from' and the 'to' address.

### PR DESCRIPTION
**Pull request by jonathanBuhler**
Fixes #80, alters test case to catch error, and removes test case from an older build.

### CODE REVIEW

File: src/model/core/EthqlBlock.ts [hunk 1]
```diff
@@ -190,7 +190,7 @@ class EthqlBlock implements EthqlBlock {
 
     // Note: the EthqlAccount#equals method is lenient to nulls/undefined.
     return this._transactions
-      .filter(tx => tx.from.equals(from) || tx.to.equals(to))
+      .filter(tx => (from == null || tx.from.equals(from)) && (to == null || tx.to.equals(to)))
```
Comment (akhila-raju) [inline src/model/core/EthqlBlock.ts]: Should be '===' not '=='

File: src/__tests__/core/block.transactionsRoles.test.ts [hunk 2]
```diff
@@ -236,8 +236,8 @@ test('block->transactionsRoles: both from and to addresses, no matching transact
 test('block->transactionsRoles: both from and to addresses, matching transactions returned', async () => {
   const query = `
     {
-      block(number: 5755715) {
-        transactionsRoles(from: "0xCcF1D27AfE45BA5F8c15265199E5b635AF4cb889", to: "0xA4eA687A2A7F29cF2dc66B39c68e4411C0D00C49") {
+      block(number: 5450945) {
+        transactionsRoles(from: "0x3f5CE5FBFe3E9af3971dD833D26bA9b5C936f0bE", to: "0x4156D3342D5c385a87D264F90653733592000581") {
```
Comment (akhila-raju) [inline src/__tests__/core/block.transactionsRoles.test.ts]: Is this an address that has sent multiple transactions on this block? If not, I'd change it to ensure that it's truly fetching the transaction with these specified 'from' and 'to' addresses.

Comment (raulk) [reply src/__tests__/core/block.transactionsRoles.test.ts]: Good call!

Comment (jonathanBuhler) [reply src/__tests__/core/block.transactionsRoles.test.ts]: It has sent multiple transactions, the original test didn't which is why the bug slipped through :)

Comment (akhila-raju) [reply src/__tests__/core/block.transactionsRoles.test.ts]: Thanks for fixing @jonathanBuhler!! :)

**Pull request merged**